lib/Server.js

```diff
@@ -737,7 +737,9 @@ class Server {
     this.hostname = hostname;
 
     return this.listeningApp.listen(port, hostname, (err) => {
-      this.createSocketServer();
+      if (this.options.hot || this.options.liveReload !== false) {
```

Comment:
```diff
- if (this.options.hot || this.options.liveReload !== false) {
+ if (this.options.hot || this.options.liveReload) {
```

Reply:
@hiroppy sorry for my late reply but I think if I changed it, it won't create the socket server if options.liveReload was `undefined` the default value. I suggest making little refactor and write `this.options.liveReload = this.options.liveReload !== false` at the beginning of the constructor and replace all `this.options.liveReload !== false` with `this.options.liveReload`.

Reply:
/cc @hiroppy

Reply:
ok, we will fix this in the next vesion.